app/components/form/WoW/PriceGroupForm/index.tsx

```
@@ -0,0 +1,257 @@
+    onChange({
+      name: newName,
+      item_ids: itemIds,
+      categories
+    })
+  }
+
+  const handleItemSelect = (value: string) => {
+    setCurrentItemName(value)
+    const itemId = getItemIDByName(value.trim(), wowItems)
+    if (itemId) {
+      const newIds = [...new Set([...itemIds, parseInt(itemId)])]
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)